code/datums/martial/boxing.dm

```diff
@@ -281,6 +296,66 @@
+	pacifist_style = FALSE
+	default_damage_type = BRUTE
+	boxing_traits = list(TRAIT_BOXING_READY)
+	/// The mobs we are looking for to pass the honor check
+	var/honorable_mob_biotypes = MOB_BEAST | MOB_SPECIAL | MOB_PLANT | MOB_BUG
+
+/datum/martial_art/boxing/hunter/honor_check(mob/living/possible_boxer)
+	// Unlike normal, this isn't an early return, because we're looking for MOB_HUMANOID as an early return. We DO want to return true for any instance where our target is boxing ready.
+	if(HAS_TRAIT(possible_boxer, TRAIT_BOXING_READY))
+		return TRUE
+
+
+	if(possible_boxer.mob_biotypes & MOB_HUMANOID && !istype(possible_boxer, /mob/living/simple_animal/hostile/megafauna)) //We're after animals, not people. Unless they want to box. (Or a megafauna)
+		return FALSE
+
+	if(possible_boxer.mob_biotypes & honorable_mob_biotypes) //We're after animals, not people
+		return TRUE
+
+	return FALSE //rather than default assume TRUE, we default assume FALSE. After all, there could be mobs that are none of our biotypes and also not humanoid. By default, they would be valid for being boxed if TRUE.
+
+// Our hunter boxer applies a rebuke and double damage against the target of their crit. If the target is humanoid, we just perform our regular crit effect instead.
+
+/datum/martial_art/boxing/hunter/crit_effect(mob/living/attacker, mob/living/defender, armor_block = 0, damage_type = STAMINA, damage = 0)
+	if(defender.mob_biotypes & MOB_HUMANOID && !istype(defender, /mob/living/simple_animal/hostile/megafauna))
+		return ..() //Applies the regular crit effect if it is a normal human, and not a megafauna
+
+	var/first_word_strike = pick("Extinction", "Brutalization", "Explosion", "Adventure", "Thunder", "Lightning", "Sonic", "Atomizing", "Whirlwind", "Tornado", "Shark", "Falcon")
+	var/second_word_strike = pick(" Punch", " Pawnch", "-punch", " Jab", " Hook", " Fist", " Uppercut", " Straight")
+	defender.visible_message(
+		span_danger("[attacker] knocks the absolute bajeezus out of [defender] utilizing the terrifying [first_word_strike][second_word_strike]!!!"),
+		span_userdanger("You have the absolute bajeezus knocked out of you by [attacker]!!!"),
+		span_hear("You hear a sickening sound of flesh hitting flesh!"),
+		COMBAT_MESSAGE_RANGE,
+		attacker,
+	)
+	to_chat(attacker, span_danger("You knock the absolute bajeezus out of [defender] out with the terrifying [first_word_strike][second_word_strike]!!!"))
+	if(ishuman(attacker))
+		var/mob/living/carbon/human/human_attacker = attacker
+		human_attacker.force_say()
+		human_attacker.say("[first_word_strike][second_word_strike]!!!", forced = "hunter boxing enthusiastic battlecry")
+	defender.apply_status_effect(/datum/status_effect/rebuked)
+	defender.apply_damage(damage * 2, default_damage_type, BODY_ZONE_CHEST, armor_block) //deals double our damage AGAIN
+	attacker.reagents.add_reagent(/datum/reagent/medicine/omnizine/godblood, 3) //Get a little healing in return for a successful crit
```

Comment:
can you theoretically overdose on pure machismo energy if you work up to 150u of this?

can godblood trigger allergies?

Reply:
I am fairly sure that, due to how poorly implemented medicine allergy is, that godsblood is actually one of the potential allergies you can have. I don't care to account for that; negative quirks are entirely a players choice thing.

You absolutely could overdose, but like...I doubt it will happen. You will need to crit some fifty times in rapid succession, and even then, you are metabolizing it.

What I could do is swap it for a bespoke chem without an overdose threshold.